rmgpy/data/kinetics/familyTest.py

```diff
@@ -40,14 +40,16 @@
 from rmgpy.data.thermo import ThermoDatabase
 from rmgpy.data.kinetics.database import KineticsDatabase
 from rmgpy.data.kinetics.family import TemplateReaction
+from rmgpy.kinetics.arrhenius import Arrhenius
 from rmgpy.data.rmg import RMGDatabase
 from rmgpy.molecule import Molecule
 from rmgpy.species import Species
-from rmgpy.molecule.group import Group
+from rmgpy.tools.loader import loadRMGPyJob
+from rmgpy.rmg.main import RMG
+from rmgpy.rmg.listener import ConcentrationPrinter
```

Comment:
Imports should be in alphabetical order if possible.

Reply:
Just to follow-up, we also break the imports into three sections: modules in the standard library, modules from our dependencies (listed in our conda environment files) and then a third block for our own modules (rmgpy + arkane).

With this, logging should be in the first section, and numpy should be in the second section (leave a blank line between sections). 

Also, I don't think that `from rmgpy.molecule.group import Group` is currently being used